aws/IaC/CDK/test-infra/config/static_config/ACM_Resources.py

```diff
@@ -0,0 +1,15 @@
+"""
+This file defines ACM resource static configuration parameters for CDK Constructs
+"""
+from aws_cdk import aws_certificatemanager
+
+ACM_Certs_List = {
+    # Pattern
+    # "cdk-id": ("domain-name", "validation-method")
+    # Example:
+
+    # "cdk-poc": (
+    #     "cdk-poc.kubeflow-testing.com",
+    #     aws_certificatemanager.ValidationMethod.DNS,
+    # )
```

Comment:
Comment out resources to serve as examples.

No effect on current test-infra yet, will send out follow up PR for further action items